calico-vpp-agent/Makefile

```diff
@@ -26,6 +26,7 @@ image: build gobgp
 	@if [ "${ALSO_LATEST}" = "y" ]; then \
 		docker tag calicovpp/agent:$(TAG) calicovpp/agent:latest; \
 	fi
+	docker image prune -f
```

Comment:
Does this have to be part of the `make image` ? 
Because it might impact non related images present on the machine

Reply:
Yes I can remove it

Reply:
thanks 😄